databases.go

```diff
@@ -50,24 +52,44 @@ const (
 	DatabaseStatusBackingUp    DatabaseStatus = "backing_up"
 )
 
+const (
+	DatabasePlatformRDBMSLegacy  DatabasePlatform = "rdbms-legacy"
+	DatabasePlatformRDBMSDefault DatabasePlatform = "rdbms-default"
+)
+
+const (
+	DatabaseMemberTypePrimary  DatabaseMemberType = "primary"
+	DatabaseMemberTypeFailover DatabaseMemberType = "failover"
+)
+
 // A Database is a instance of Linode Managed Databases
 type Database struct {
-	ID              int            `json:"id"`
-	Status          DatabaseStatus `json:"status"`
-	Label           string         `json:"label"`
-	Hosts           DatabaseHost   `json:"hosts"`
-	Region          string         `json:"region"`
-	Type            string         `json:"type"`
-	Engine          string         `json:"engine"`
-	Version         string         `json:"version"`
-	ClusterSize     int            `json:"cluster_size"`
-	ReplicationType string         `json:"replication_type"`
-	SSLConnection   bool           `json:"ssl_connection"`
-	Encrypted       bool           `json:"encrypted"`
-	AllowList       []string       `json:"allow_list"`
-	InstanceURI     string         `json:"instance_uri"`
-	Created         *time.Time     `json:"-"`
-	Updated         *time.Time     `json:"-"`
+	ID          int              `json:"id"`
+	Status      DatabaseStatus   `json:"status"`
+	Label       string           `json:"label"`
+	Hosts       DatabaseHost     `json:"hosts"`
+	Region      string           `json:"region"`
+	Type        string           `json:"type"`
+	Engine      string           `json:"engine"`
+	Version     string           `json:"version"`
+	ClusterSize int              `json:"cluster_size"`
+	Platform    DatabasePlatform `json:"platform"`
+	Fork        *Database        `json:"fork,omitempty"`
```

Comment:
Do we include `Fork` in any request? If not, I think we may remove `omitempty` tag?

Reply:
The `Database` struct is only used in `ListDatabases`.

Reply:
So do you think it would make sense to remove the `omitempty` tags?

Reply:
Since we do use the struct, I don't see any harm in keeping them. What do you think?